src/work-allocation-2/models/cases/index.ts

```diff
@@ -0,0 +1,13 @@
+import InvokedCaseAction from './invoked-case-action.model';
+import CaseAction from './case-action.model';
+import CaseFieldConfig from './case-field-config.model';
+import CaseServiceConfig from './case-service-config.model';
+import Case from './case.model';
+
+export {
+  InvokedCaseAction,
```

Comment:
_Nitpick:_
Ordering here and in the imports?

Reply:
Ordered